src/sqlfluff/rules/ambiguous/AM08.py

```diff
@@ -0,0 +1,95 @@
+"""Implementation of Rule AM08."""
+
+from typing import Optional, Tuple
+
+from sqlfluff.core.parser import (
+    ComparisonOperatorSegment,
+    KeywordSegment,
+    LiteralSegment,
+    WhitespaceSegment,
+)
+from sqlfluff.core.rules import BaseRule, LintFix, LintResult, RuleContext
+from sqlfluff.core.rules.crawlers import SegmentSeekerCrawler
+
+
+class Rule_AM08(BaseRule):
+    """Join clauses should be present.
+
```

Comment:
I think there should be a more more preamble in the documentation to explain _why_ this rule exists. Something like:
> cross joins are valid, but rare in the wild - and more often created _by mistake_ than on purpose. This rule catches situations where a cross join has been specified, but not explicitly and so the risk of a mistaken cross join is highly likely.

Reply:
That will obviously need rewording, but I think you get the idea of what I mean.

Reply:
Sounds good to me, I'll fix.